lib/Model/BinaryContainer.cpp

```diff
@@ -0,0 +1,11 @@
+/*
+ * This file is distributed under the MIT License. See LICENSE.md for details.
+ */
+
+#include "revng/Model/BinaryContainer.h"
+
+using namespace llvm;
+using namespace std;
+using namespace Model;
```

Comment:
Weird?

Reply:
It's just a predule I put in almost every cpp file. indeed it's not needed toh. Does it have to go?

Reply:
Yeah.